bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Title.java

```diff
@@ -62,12 +65,13 @@ default String getType() {
     }
 
     /**
-     * Returns the Title's link URL, if one was set.
+     * Returns the Title's link, if one was set.
      *
-     * @return the title's link URL, if one was set, or {@code null}
-     * @since com.adobe.cq.wcm.core.components.models 12.4.0
+     * @return the title's link, if one was set, or {@code null}
+     * @since com.adobe.cq.wcm.core.components.models 13.0.0
      */
-    default String getLinkURL() {
+    @Nullable
```

Comment:
We need to keep backwards compatiblity. Instead of replacing this method with `getLink` we should keep it, mark it deprecated and make sure it defers to `getLink` in the implementation.

Reply:
`getLink` should not be introduced in Title v1. It will be introduced in Title v3:
https://github.com/adobe/aem-core-wcm-components/pull/1419

Reply:
I think it's ok to introduce `getLink` in v2 as well as long as we also keep the old `getLinkURL` method. We already did that in https://github.com/adobe/aem-core-wcm-components/blob/master/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/ListItem.java#L33-L42

Reply:
IIRC we had decided to introduce the new `getLink` API and use it in HTL templates in a new component version e.g. #1419 ) to avoid any backwards compatibility issue. Am I missing something?

Reply:
I think we can keep backwards compatibility if we only use it to get the new accessibility attributes and use the old/deprecated method for the `href`. It will create some merge conflicts with #1419 but that can be addressed afterwards.

I see no point in adding the new attributes to the title model when we know the proper way forward is to get them via `Link`.

Reply:
>I think we can keep backwards compatibility if we only use it to get the new accessibility attributes
How would you do that as there is no API to get only the 2 accessibility properties?

Reply:
Answering to myself: we could instantiate the link object with only the  2 accessibility properties, the other ones being `null`. A bit hacky but I guess it would work until we introduce the new version of the Title component that supports the link handler.

Reply:
@jckautzmann , we should instantiate the Link as usual, continue to get the `href` from `getLinkURL` (but change the implementation there to get the value from `Link#getUrl`) and inject the rest of the attributes like: `aria-label="${title.link.htmlAttributes['aria-label']}"`.